package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tte-api-services",
-  "version": "1.0.6",
+  "version": "1.1.0",
```

Comment:
The existing version is 1.1.0, not sure how it's possible that you set this version now

Reply:
Version should be 1.2.0

Reply:
done